guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc

```diff
@@ -46,7 +46,7 @@ To create a suitable Puppet environment to be associated with a host group, foll
 ----
 . In the {ProjectWebUI}, navigate to *Configure* > *Puppet ENC* > *Environments*.
 . Click *Import environment from*.
-The button name includes the FQDN of the internal or external {SmartProxy}.
+The button name includes the FQDN of your {ProjectServer} and {SmartProxyServers}.
```

Comment:
I think button name is a poor description. From memory the way it works is:
* If there's 1 Smart Proxy, it has the Smart Proxy **name** (not FQDN - we just happen to set the FQDN as the name)
* If there are multiple, it's a drop down with all the Smart Proxy names

I'd prefer something that's closer to:
```suggestion
. Select your {SmartProxy}
```

Reply:
I agree and like your suggestion -> applied.